docs/qute/TemplateSupport.md

```diff
@@ -0,0 +1,93 @@
+# Qute Template support
+
+Qute support provides `completions`, `validation`, `hover`, `definition`, `highlight`, `linked editing` in Qute template.
+
+To benefit with this support, we recommend your template files to follow the `*.qute.*` naming pattern (ex: `items.qute.html`, `items.qute.txt`, `items.qute.yaml`, etc.).
+
+## Parameter declaration support
+
+A `parameter declaration` is used to declare the expected model in a Qute template:
+
+```

```

Comment:
maybe add 'html' here, and other qute snippets to show same syntax highlighting

Reply:
The parameter declaration is a Qute syntax not an html syntax, so we cannot higlight Qute syntax, but perhaps it should b ereally interesting to reference our Qute syntax inside github. It seems that we should create a PR at https://github.com/github/linguist @fbricon what do you think about this idea?

Reply:
+1